pipeline_dp/histograms.py

```diff
@@ -230,6 +262,30 @@ def _compute_l0_contributions_histogram(
                                         HistogramType.L0_CONTRIBUTIONS)
 
 
+def _compute_l1_contributions_histogram(
+        col, backend: pipeline_backend.PipelineBackend):
+    """Computes histogram of the number of distinct records contributed by a privacy id.
+    This histogram contains: number of privacy ids which contributes to 1
```

Comment:
Optional: "which contributes 1 record, 2 records, etc."`

Also, what do we do in the case of sum?

Reply:
This l1 contribution histogram is about number of total contributions, which can be used for estimating l1 sensitivity bounds for sum as well.

Reply:
What is num_contributions when a user contributes -1,+1 across different partitions in the case of sum? Is it 0?

Reply:
There are multiple ways how to make l1 contribution bounding. The easiest is that
`l1_bound = num_contributions*max(abs(min_value), abs(max_value))`
And for such contribution bounding `num_contributions` is needed